doc/sphinx/Pacemaker_Administration/pcs-crmsh.rst

```diff
@@ -0,0 +1,437 @@
+
+.. topic:: Create a Location Constraint
+
+   .. code-block:: none
+
+      crmsh # crm configure location prefer-pcmk-1 WebSite 50: pcmk-1
+      pcs   # pcs constraint location WebSite prefers pcmk-1=50
+    
+.. topic:: Create a Location Constraint Based on Role
+
+   .. code-block:: none
+
+      crmsh # crm configure location prefer-pcmk-1 WebSite rule role=Promoted 50: \#uname eq pcmk-1
+      pcs   # pcs constraint location WebSite rule role=Promoted 50 \#uname eq pcmk-1
+
+.. topic:: Move a Resource to a Specific Node (by Creating a Location Constraint)
+
+   .. code-block:: none
+
+      crmsh     # crm resource move WebSite pcmk-1
+      pcs       # pcs resource move WebSite pcmk-1
+      pacemaker # crm_resource -r WebSite --move -N pcmk-1
+    
+.. topic:: Move a Resource Away from Its Current Node (by Creating a Location Constraint)
+
+   .. code-block:: none
+
+      crmsh     # crm resource ban Website pcmk-2
+      pcs       # pcs resource ban Website pcmk-2
+      pacemaker # crm_resource -r WebSite --move
+
+.. topic:: Remove any Constraints Created by Moving a Resource
+
+   .. code-block:: none
+
+      crmsh     # crm resource unmove WebSite
+      pcs       # pcs resource clear WebSite
+      pacemaker # crm_resource -r WebSite --clear
+
+Advanced Configuration
+######################
+
+Manipulate Configuration Elements by Type
+_________________________________________
+
+.. topic:: List Constraints with IDs
+
+   .. code-block:: none
+
+      pcs   # pcs constraint list --full
+
+.. topic:: Remove Constraint by ID
+
+   .. code-block:: none
+
+      pcs   # pcs constraint remove cli-ban-Website-on-pcmk-1
+      crmsh # crm configure remove cli-ban-Website-on-pcmk-1
+
+crmsh's `show` and `edit` commands can be used to manage resources and
+constraints by type:
+
+.. topic:: Show Configuration Elements
+
+   .. code-block:: none
+
+      crmsh # crm configure show type:primitive
+      crmsh # crm configure edit type:colocation
+
+Batch Changes
+_____________
+
+.. topic:: Make Multiple Changes and Apply Together
+
+   .. code-block:: none
+
+      crmsh # crm
+      crmsh # cib new drbd_cfg
+      crmsh # configure primitive WebData ocf:linbit:drbd params drbd_resource=wwwdata \
+              op monitor interval=60s
+      crmsh # configure ms WebDataClone WebData meta master-max=1 master-node-max=1 \
+              clone-max=2 clone-node-max=1 notify=true
+      crmsh # cib commit drbd_cfg
+      crmsh # quit
+
+      pcs      # pcs cluster cib drbd_cfg
+      pcs      # pcs -f drbd_cfg resource create WebData ocf:linbit:drbd drbd_resource=wwwdata \
+                 op monitor interval=60s
+      pcs-0.9  # pcs -f drbd_cfg resource master WebDataClone WebData master-max=1 master-node-max=1 \
+                 clone-max=2 clone-node-max=1 notify=true
+      pcs-0.10 # pcs -f drbd_cfg resource promotable WebData promoted-max=1 promoted-node-max=1 \
```

Comment:
```suggestion
      pcs-0.10 # pcs -f drbd_cfg resource promotable WebData WebDataClone promoted-max=1 promoted-node-max=1 \
```
Custom clone names are supported since pcs-0.10.7. Yes, the order is the other way around compared to pcs-0.9.